### DISCUSSION TITLE
insert 插入数据时候的流程是什么样的呀，想知道副本同步的机制详情

### DISCUSSION THREAD

**Discussion by y3k-inf** [apache/doris#12799] [A - General / Q&A]
insert 插入数据时候的流程是什么样的呀，想知道副本同步的机制详情

**Comment by LemonLiTree** [comment]
https://doris.apache.org/zh-CN/docs/dev/admin-manual/maint-monitor/tablet-repair-and-balance

**Reply by y3k-inf** [reply]
我好奇的是在写入数据的时候是如何做数据同步的，这个是副本的后台管理策略。是写入的时候都只会写一份数据，副本通过管理策略来进行同步吗？
